docker/opendjsetup.sh

```diff
@@ -13,13 +12,13 @@ if [ -n "${NUMBER_SAMPLE_USERS+set}" ]; then
 fi
 
 /opt/opendj/setup directory-server --ldapPort 389 \
+  --instancePath ${INSTANCE_ROOT} \
```

Comment:
how does this get the value for ${INSTANCE_ROOT}? I see that env variable defined in the firecloud-opendj repo but not in this repo.

Reply:
another question: why do we need to add `instancePath` here?

Reply:
`instancePath` seems to have not been default in the GCR dockers? It may be that there was a command getting run in our previous image that we are missing here, but figuring out is basically impossible at this point.

${INSTANCE_ROOT} is defined at the dockerfile level here https://github.com/broadinstitute/firecloud-opendj/blob/dev/Dockerfile#L54, so it will be on the docker container's environment variables, which means that this will get picked up with when run on the docker container.

Reply:
oh I see, this file is copied into the docker container (https://github.com/broadinstitute/sam/blob/c34ff18ddb109aa15e4f4cade8377f582414c86c/docker/stand-alone/docker-compose.yaml#L33) and is run within that container, so that makes sense.